daphne_worker/src/config.rs

```diff
@@ -319,15 +319,11 @@ impl DaphneWorkerConfig {
     }
 }
 
-/// Daphne-Worker state, used for handling a DAP request. Includes long-lived configuration,
-/// metrics, cached responses from KV, etc.
-pub(crate) struct DaphneWorkerState {
+/// Daphne-Worker shared state, which may be used by multiple requests. Includes long-lived configuration,
+/// cached responses from KV, etc.
+pub(crate) struct DaphneWorkerSharedState {
```

Comment:
* Perhaps "Isolate" state instead of "Shared" state?
* Random thought here: If we do a "hotfix" of data stored in KV in a live deployment, we would also want to tear down any isolates that happen to be running so they can pick up the changes. If we do "wrangler publish", will this restart any isolates that are running?

Reply:
I switched to "isolate".  I don't know if wrangler publisher will tear down old stuff, and even if it did I'm not sure it would be properly synchronized.